src/ui/articlesaver.cc

```diff
@@ -214,45 +213,48 @@ void ArticleSaver::save()
         filterAndCollectResources( html, rx1, "\"", folder, resourceIncluded, downloadResources );
         filterAndCollectResources( html, rx2, "'", folder, resourceIncluded, downloadResources );
 
-        auto * progressDialog    = new ArticleSaveProgressDialog( uiParent_ );
-        int maxVal               = 1; // main html
-        bool anyHandlerConnected = false;
+        int totalResources         = downloadResources.size();
+        auto completedResourcesPtr = std::make_shared< int >( 0 );
+
+        if ( totalResources > 0 ) {
+          emit statusMessage( QObject::tr( "Saving article... (0/%1)" ).arg( totalResources ), 0 );
+        }
+        else {
+          emit statusMessage( QObject::tr( "Saving article..." ), 0 );
+        }
```

Comment:
start message is not needed .  Users will have no chance(time) to see it.

IMO ,it can be removed.